polymer/src/elements/reportable-meta.html

```diff
@@ -124,14 +151,40 @@
 
       _handleInput: function () {
         var field = Polymer.dom(event).rootTarget;
+        var narrativeTextInput = Polymer.dom(this.root).querySelectorAll('labelled-item')[1].querySelector('input');
```

Comment:
You can use `this.$` notation to access DOM element if it has id attribute. Since `input` tag has `narrative_assessment`, you should be able to access via `this.$`
```suggestion
        var narrativeTextInput = this.$.narrative_assessment;
```

Reply:
Just tried this, but it comes back `undefined`. I think it's because the element is inside a `dom-if` template?

Reply:
Yeah I think you're right. this.$ only works at local static DOM elements. I did not see this element was under template `dom-if` which was hidden by this PR code.

Reply:
👍 👍